src/client/components/ProductDetail/ProductDetail.styles.css

```diff
@@ -0,0 +1,125 @@
+.product-detail p {
+  font-size: 1.1rem;
+}
+
+.product-detail h3 {
+  color: #687808;
+}
+
+.product-detail {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-flow: column nowrap;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  color: black;
+}
+
+.product-detail-container {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  align-items: center;
+  flex: 1;
+  height: 100%;
+  border: 3px solid rgb(194, 194, 194);
+  padding: 2rem;
+}
+
+.image-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.image-container img {
+  max-width: 280px;
```

Comment:
adding height: 230px; might make the classes image-container and info-container of equal height!